src/test/com/gd/twitterstreamingtToDatalake/TransformTweetsTest.scala

```diff
@@ -1,30 +1,31 @@
 package com.gd.twitterstreamingtToDatalake
 
 import com.holdenkarau.spark.testing.StreamingSuiteBase
-import org.scalatest.{FunSuite, WordSpec}
-
+import org.scalatest.{FunSuite}
 import twitter4j.Status
 import org.scalamock.scalatest.MockFactory
 
-class TransformTweetsTest extends FunSuite with StreamingSuiteBase with MockFactory{
+class TransformTweetsTest extends FunSuite with StreamingSuiteBase with MockFactory {
 
-  val mockedStatus:Status = mock[twitter4j.Status]
+  test("Should Retreive Only Text from the Tweets Status") {
+    val mockedStatus: Status = mock[twitter4j.Status]
+    (mockedStatus.getText _).expects().returning("This is the #Sample #text for #test purpose")
 
-  var input = new List[mockedStatus]
+    val input = List(List(mockedStatus))
+    val expected = List(List("This is the #Sample #text for #test purpose"))
+    testOperation(input, TransformTweets.getText _, expected, ordered = false)
+  }
 
-    test("Filter only words Starting with #")  {
-      val input = List(List("#firstHash","11","22"), List("this","is","#secondHash"), List("#thirdHash","44","55"))
-      val expected = List(List("#firstHash"),List("#secondHash"),List("#thirdHash"))
+  test("Should Filter the HashTags from the Text Messages") {
+    val input = List(List("#firstHash", "11", "22"), List("this", "is", "#secondHash"), List("#thirdHash", "44", "55"))
+    val expected = List(List("#firstHash"), List("#secondHash"), List("#thirdHash"))
 
-      testOperation(input, TransformTweets.getHashTags _, expected, ordered = false)
-    }
+    testOperation(input, TransformTweets.getHashTags _, expected, ordered = false)
+  }
 
-  test("really simple transformation") {
+  test("Sample Transformation to check the StreamingTestSuite") {
```

Comment:
Could you please clarify why you think it's important to check `StreamingTestSuite` itself? Does the check relate to the implemented business logic?

Reply:
This was for personal debug.
Removed from the code..